rhc-ose-ansible/inventory/ose-provision

```diff
@@ -12,6 +12,33 @@ openstack_key_name=""
 #openshift_openstack_master_storage_size: 10
 #openshift_openstack_node_storage_size: 10
 
+#env_id=
+rhc_ose_inv_dest=.
+rhc_ose_cluster_method=native
```

Comment:
These are additions from the openshift-install PR. I used the same inventory file from the openshift-install role, since you would be using the same inventory file, as this would all be a part of the same play?